src/angular-wysiwyg.js

```diff
@@ -337,6 +342,15 @@ Requires:
                 scope.setHiliteColor = function() {
                     scope.format('hiliteColor', scope.hiliteColor)
                 }
+                
+                scope.setCssClass= function() {
+                    var classes = scope.cssClasses.join(' ');
+                    var selection = window.getSelection().focusNode.parentNode;
+                    $(selection).removeClass(classes);
+                    if(scope.cssClass !== 'css') {
+                      $(selection).addClass(scope.cssClass);
+                    }
+                };
```

Comment:
Wouldn't it make more sense to just add the selected css class to the highlighted text or starting where the cursor is like the Bold, Italic, etc instead of just the parent node.  This might work for you but may not be suitable everyone else.